components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -863,26 +887,24 @@ def start_webui(instance_id: str, clp_config: CLPConfig, mounts: CLPDockerMounts
         "--network", "host",
         "--name", container_name,
         "--log-driver", "local",
-        "-e", f"NODE_PATH={node_path}",
-        "-e", f"MONGO_URL={clp_config.results_cache.get_uri()}",
-        "-e", f"PORT={clp_config.webui.port}",
-        "-e", f"ROOT_URL=http://{clp_config.webui.host}",
-        "-e", f"METEOR_SETTINGS={json.dumps(meteor_settings)}",
-        "-e", f"CLP_DB_USER={clp_config.database.username}",
-        "-e", f"CLP_DB_PASS={clp_config.database.password}",
-        "-e", f"WEBUI_LOGS_DIR={container_webui_logs_dir}",
-        "-e", f"WEBUI_LOGGING_LEVEL={clp_config.webui.logging_level}",
         "-u", f"{os.getuid()}:{os.getgid()}",
     ]
     # fmt: on
+    env_vars = [
+        f"NODE_PATH={node_path}",
+        f"MONGO_URL={clp_config.results_cache.get_uri()}",
+        f"PORT={clp_config.webui.port}",
+        f"ROOT_URL=http://{clp_config.webui.host}",
+        f"METEOR_SETTINGS={json.dumps(meteor_settings)}",
+        f"CLP_DB_USER={clp_config.database.username}",
+        f"CLP_DB_PASS={clp_config.database.password}",
+        f"WEBUI_LOGS_DIR={container_webui_logs_dir}",
```

Comment:
if the removal isn't intentional
```suggestion
        f"WEBUI_LOGS_DIR={container_webui_logs_dir}",
        f"WEBUI_LOGGING_LEVEL={clp_config.webui.logging_level}",
```